### DISCUSSION TITLE
Hpool

### DISCUSSION THREAD

**Discussion by Neppumuck** [ericaltendorf/plotman#625] [Q&A]
Hi,
i try to create plots for hpool. when i add my keys in the plotman.yaml, i get this:

`marshmallow.exceptions.ValidationError: {'plotting': {'pool_pk': ['Not a valid string.'], 'farmer_pk': ['Not a valid string.']}}`

i know, the key from hpool is longer than ciha itself. example: 0xa0c362bfb4a410181fd3c3151d84346fc65113df668e4f8fe470479d6431585fd7e1af392c09b0b091c3e8c7b54f427c
8bb4ed2a20ad1a134a293c80b65384159f7ff38509f697cc1277d7a91f617187b8e6b7179392ad7763fd9cafc14ad9a5

so i had to remove simply the 0x in the front i can use plotman for hpool plots?

greetings
Neppu

**Comment by Purple-Tentacle** [comment]
I would advise to not use hpool.
It is closed source and nobody knows what is going on. 
Wait for the official Chia pooling, expected to be released end of May. 
And yes, plots must be plotted again, but with the new ones you can farm on your own or change the pool.
